File: BAEKJOON/C/2576.c
#include <stdio.h>

int main () {
    int sum = 0, min = 0, i, input;
    
    for(i = 0; i < 7; i++){
        scanf("%d", &input);
        if(min == 0 && input % 2 == 1){
            min = input;
            sum += input;
        }
        
        else if(input % 2 == 1){
            sum += input;
            min = min < input ? min : input;
        }
    }
    if(sum == 0){
        printf("%d", -1);
    }
    else{
        printf("%d\n%d", sum, min);
    }
}//O(1)

File: 2학년1학기/BAEKJOON/python/2576.py
import sys

odd_arr = []

for i in range(7):
    num = int(sys.stdin.readline().rstrip())
    if num % 2 == 1:
        odd_arr.append(num)

if not len(odd_arr):
    print(-1)
else:
    print(sum(odd_arr))
    print(min(odd_arr))
#O(1)?